### ISSUE TITLE
Error : no attribute 'ParamSpec'

### ISSUE THREAD

**Issue by tomitomas** [KaSroka/Toshiba-AC-control#58]
Hello,

I'm hitting an issue when I try to start to use the lib toshiba-ac v0.3.7 on Python 3.9.2

```
File "resources/venv/lib/python3.9/site-packages/toshiba_ac/device_manager.py", line 19, in <module>
from toshiba_ac.device import ToshibaAcDevice
File "resources/venv/lib/python3.9/site-packages/toshiba_ac/device/__init__.py", line 23, in <module>
from toshiba_ac.device.features import ToshibaAcFeatures
File "resources/venv/lib/python3.9/site-packages/toshiba_ac/device/features.py", line 32, in <module>
from toshiba_ac.utils import pretty_enum_name
File "resources/venv/lib/python3.9/site-packages/toshiba_ac/utils/__init__.py", line 51, in <module>
P = t.ParamSpec("P")
AttributeError: module 'typing' has no attribute 'ParamSpec'
```

any idea ?

**Comment by tomitomas** [comment]
packages installed : 
```
aiohappyeyeballs==2.4.0
aiohttp==3.10.5
aiosignal==1.3.1
async-timeout==4.0.3
attrs==24.2.0
azure-iot-device==2.14.0
certifi==2024.7.4
charset-normalizer==3.3.2
deprecation==2.1.0
frozenlist==1.4.1
idna==3.8
janus==1.0.0
multidict==6.0.5
packaging==24.1
paho-mqtt==1.6.1
pkg_resources==0.0.0
PySocks==1.7.1
requests==2.32.3
requests-unixsocket2==0.4.1
toshiba-ac==0.3.7
typing_extensions==4.12.2
urllib3==2.2.2
yarl==1.9.4
```

**Comment by tomitomas** [comment]
Any idea @KaSroka !?

**Comment by KaSroka** [comment]
Good catch, since v0.3.7 python 3.10 or newer is required. I forgot to update the setup.cfg

**Comment by tomitomas** [comment]
looks OK pour python 3.10
thanks !

**Comment by tomitomas** [comment]
hi @KaSroka 

looks like you still didnt update setup.cfg with the newest release you've just done ;)

**Comment by tomitomas** [comment]
fix : e8d5bb7
